pydgraph/client_slash.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2020 Dgraph Labs, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Slash GraphQL python client."""
+import grpc
+
+import pydgraph
+from pydgraph import txn, util, DgraphClientStub
+from pydgraph.meta import VERSION
+from pydgraph.proto import api_pb2 as api
+
+__author__ = 'Anurag Sharma <anurag@dgraph.io>'
+__maintainer__ = 'Martin Martinez Rivera <martinmr@dgraph.io>'
+__version__ = VERSION
+__status__ = 'development'
+
+
+class SlashGraphQLClient(object):
+    """Creates a new Client for interacting with the Slash GraphQL.
+    """
+
+    def __init__(self, SlashEndpoint, APIKey):
```

Comment:
generally, we use snake casing for variables in python - use slash_end_point and api_key instead.